src/cpp/session/modules/SessionProfiler.R

```diff
@@ -13,4 +13,54 @@
 #
 #
 
+.rs.addJsonRpcHandler("start_profiling", function(profilerOptions)
+{
+   tryCatch({
+      tempPath <- .Call("rs_profilesPath")
+      if (!dir.exists(tempPath)) {
```

Comment:
Yes, that's usually the easiest way to find problems. We officially support R 2.11.1 and newer (and even then people occasionally ask us for a build that supports an older version!). It's been a long time since we bumped the required version, though, so we could make an argument for bumping the R version requirement in this release, although that's irrelevant to `dir.exists` since it's too new.